CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img width="60%" src="https://raw.githubusercontent.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/master/docs/data/MIVisionX.png" /></p>
+<p align="center"><img width="60%" src="https://raw.githubusercontent.com/GPUOpen-ProfessionalCompute-Libraries/mivisionx/develop/docs/data/MIVisionX.png" /></p>
```

Comment:
@samjwu Is this meant to be all small letters for "mivisionx"? The repo is still "MIVisionX"

Reply:
The URL should still work since it's case-insensitive, but I have fixed this in [eabf50d](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pull/1072/commits/eabf50d3d74169ac823dab97a3a4f0090c703797).